lib/swaps/Swaps.ts

```diff
@@ -231,27 +268,37 @@ export class Swaps extends EventEmitter {
    */
   private handleDealResponse = (responsePacket: packets.DealResponsePacket, peer: Peer): void  => {
     const responseBody = responsePacket.body!;
-    const deal = this.getDeal(SwapDealRole.Taker, responseBody.takerDealId);
+    this.logger.debug(`Received swap request: ${JSON.stringify(responseBody)}`);
```

Comment:
I would put this message in `Pool.ts` and print something like `received swap request from ${peer.nodePubKey}: ${JSON.stringify(responseBody)}` to stay consistent with the other messages in the pool